plugins/source/gcp/resources/services/resourcemanager/organization_tag_values.go

```diff
@@ -0,0 +1,57 @@
+package resourcemanager
+
+import (
+	resourcemanager "cloud.google.com/go/resourcemanager/apiv3"
+	pb "cloud.google.com/go/resourcemanager/apiv3/resourcemanagerpb"
+	"context"
+	"github.com/apache/arrow/go/v14/arrow"
+	"github.com/cloudquery/cloudquery/plugins/source/gcp/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+	"google.golang.org/api/iterator"
+)
+
+func OrganizationTagValues() *schema.Table {
```

Comment:
Can we rename this function to `organizationTagValues` so it becomes a private one? We usually do it for non top level tables so they don't get mistakenly used outside their package

Reply:
Made dependent table methods private.